lib/pacemaker/pcmk_sched_location.c

```diff
@@ -175,24 +235,22 @@ generate_location_rule(pcmk_resource_t *rsc, xmlNode *rule_xml,
             return false;
     }
 
+    /* Users may configure the rule with either a score or the name of a
+     * node attribute whose value should be used as the constraint score for
+     * that node.
+     */
+    score_attr = score_attribute_name(rule_xml, &local_score_attr, rule_input);
```

Comment:
Slight change that's probably fine: previously, the score took precedence, while now the score attribute takes precedence. Both may be present if validation is disabled.

Reply:
Right, since schema validation prevents it, I wasn't concerned about the change in behavior